src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+@use '../_variables/variables' as *;
+
+.card {
+  width: $CardWidth;
+  padding: $CardWidth * 0.01 * 16 $CardWidth * 0.01 * 8 $CardWidth * 0.01 * 8;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: $CardWidth * 0.8;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: $FontMedium;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $ColorSecondary;
+    margin-bottom: 16px;
+  }
+
+  &__rank {
+    display: flex;
+    justify-content: space-between;
+    align-items: stretch;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $ColorMainAccent;
```

Comment:
The variable `$ColorMainAccent` should be defined in the `variables.scss` file. Ensure that it is correctly defined and imported.